classes/class-wc-connect-paypal-ec.php

```diff
@@ -184,13 +184,13 @@ public function maybe_show_banner() {
 					&& 'post' === $screen->base
 					&& 'ppec_paypal' === WC_Connect_Compatibility::instance()->get_payment_method( wc_get_order() )
 					)
-				|| ( // WooCommerce settings.
+				|| ( // WooCommerce payments settings.
 					'woocommerce_page_wc-settings' === $screen->base
 					&& isset( $_GET['tab'] ) && 'checkout' === $_GET['tab']
 					)
-				|| ( // WooCommerce payment gateway extension page
+				|| ( // WooCommerce payment gateway extension page.
 					'woocommerce_page_wc-addons' === $screen->base
-					&& isset( $_GET['section'] ) && 'payment_gateways' === $_GET['section']
+					&& isset( $_GET['section'] ) && 'payment-gateways' === $_GET['section']
```

Comment:
Nice catch. 👍

Reply:
Yeah, I'm pretty sure it changed at some point on WC side, which can happen at any time since it loads those from a remote endpoint.